cli/src/commands/prev.rs

```diff
@@ -79,11 +82,17 @@ pub(crate) fn cmd_prev(
     // If we're editing, start at the working-copy commit. Otherwise, start from
     // its direct parent(s).
     let target_revset = if edit {
-        RevsetExpression::commit(current_wc_id.clone()).ancestors_at(args.offset)
-    } else {
         RevsetExpression::commit(current_wc_id.clone())
-            .parents()
-            .ancestors_at(args.offset)
+    } else {
+        RevsetExpression::commit(current_wc_id.clone()).parents()
+    };
+    let target_revset = if args.conflict {
+        target_revset
+            .ancestors()
+            .filtered(RevsetFilterPredicate::HasConflict)
+            .roots()
```

Comment:
Should be `.heads()`?

Reply:
with `heads()` we choose the last child of the conflicted commit, instead of it, I don't think that's a improvement. I did test it locally too.

Reply:
I'm confused why you think it's expected for `jj prev --conflict` to go to the earliest possible commit (that's what you're saying, right?). Wouldn't that be some `jj prev --conflict --go-as-far-as-possible` version? Regular `jj prev` doesn't go to the root, of course.

Reply:
> I'm confused why you think it's expected for `jj prev --conflict` to go to the earliest possible commit (that's what you're saying, right?).

Yes, I kinda expect to resolve the conflict in the child and then squash into the conflicted parent (which also is our suggestion to users).  

> Wouldn't that be some `jj prev --conflict --go-as-far-as-possible` version?

Maybe? There are imo no clear semantics for it.

Reply:
> Yes, I kinda expect to resolve the conflict in the child and then squash into the conflicted parent (which also is our suggestion to users).

I agree, but I don't think we should tie the behavior of `jj prev --conflict` to that workflow. I think it will be quite unexpected that `jj next --conflict` takes you to the next conflict but `jj prev --conflict` takes you to the *first* conflict.

Reply:
> I think it will be quite unexpected that `jj next --conflict` takes you to the next conflict but `jj prev --conflict` takes you to the _first_ conflict.

`jj next --conflict` always takes you to the conflicting commit which is the _source_ of the conflict, while `jj prev --conflict` does not, which is rather inconsistent and not very symmetrical.

Reply:
Okay, let's wait to hear from others.

Reply:
I agree that going to the head of the conflicting region isn't useful, but I expect `jj next --conflict && jj prev --conflict` brings me to the original position if it's conflict. (I don't know if `jj next --conflict` in that situation is useful, though.)

FWIW, the query `ancestors(..) & conflict()` is super slow in mid-size repo.

Reply:
I kept the `heads()` version for now, but added a comment to consider my option if it is requested.